src/backend/looped/self/off_radar.cpp

```diff
@@ -20,5 +20,5 @@ namespace big
 		}
 	};
 
-	off_radar g_off_radar("otr", "Off Radar", "Hides your blip from other players", g.self.off_radar);
+	off_radar g_off_radar("otr", "Off the Radar", "Hides your blip from other players", g.self.off_radar);
```

Comment:
Capitalize "the"

Reply:
"Do not capitalize an article (a, an, the) unless it is first or last in the title. Do not capitalize a coordinating conjunction (and, or, nor, but, for, yet, so) unless it is first or last in the title."

Reply:
No fix needed as this PR will only be merged after the translation service gets added. Which will generalize this text anyways.